circle.yml

```diff
@@ -1049,6 +1097,16 @@ linux-workflow: &linux-workflow
         requires:
           - build-npm-package
           - build-binary
+    - test-binary-as-non-root-user:
+        executor: non-root-docker-user
```

Comment:
if you're going to have `test-binary-as-non-root-user` have a name without spaces, make this one have one too for consistency:

```
name: "test binary as non-root user"
```

Reply:
yup, renamed both for clarity like this:
```yaml
- test-binary-as-specific-user:
        name: "test binary as a non-root user"
        executor: non-root-docker-user
        requires:
          - build-npm-package
          - build-binary
- test-binary-as-specific-user:
        name: "test binary as a root user"
        requires:
          - build-npm-package
          - build-binary
```